user.js

```diff
@@ -32,6 +32,13 @@ user_pref("dom.webaudio.enabled",				false);
 // https://www.mozilla.org/en-US/firefox/geolocation/
 user_pref("geo.enabled",					false);
 
+// PREF: When geolocation is enabled, use Mozilla geolocation service instead of Google
+// https://bugzilla.mozilla.org/show_bug.cgi?id=689252
+user_pref("geo.wifi.uri", "https://location.services.mozilla.com/v1/geolocate?key=test");
```

Comment:
Instead of `key=test`, we may as well use the built-in Mozilla Location Service key:

    https://location.services.mozilla.com/v1/geolocate?key=%MOZILLA_API_KEY%

It's more likely to work reliably.